action.yml

```diff
@@ -25,6 +25,8 @@ runs:
   steps:
       - name: Set up Docker buildx
         uses: docker/setup-buildx-action@d70bba72b1f3fd22344832f00baa16ece964efeb # v3.3.0
+        with:
+          use: 'false'
```

Comment:
What does this do?

Reply:
@EduarteXD I see this:

https://github.com/docker/setup-buildx-action/blob/5138f76647652447004da686b2411557eaf65f33/README.md#L97

![Screenshot 2024-05-21 at 09 22 43@2x](https://github.com/replicate/setup-cog/assets/2289/e1094f81-be09-44cf-b75b-ca29dacb03e5)

.. but I don't know enough about how Docker or buildx works to know what "Switch to this builder instance" means or what effect it might have.

Reply:
In buildx, the ```--use``` parameter is used to specify an existing builder. 

In the setup-buildx-action script, passing ```use: true``` will make each build step use a new builder instance, each builder has its own context, cache, and storage space. 

Therefore, the artifacts (such as intermediate layer cache, final images, etc.) between different builders are isolated by default and cannot directly access each other’s artifacts. 

In each step of the cog's build, it is necessary to use ```--load``` to directly load the built image into the Docker engine, which can take a considerable amount of time.

Reply:
Gotcha! Thanks for explaining in detail. I can set up a test workflow that combines this change plus https://github.com/replicate/cog/pull/1656 and see how it goes. If everything is smooth I'll merge and ship these changes.